### PR TITLE
[SDP-1151] Update single-tenant to multi-tenant migration guide

### PR DESCRIPTION
**Pull request by marcelosalloum**
### What

Update the single-tenant to multi-tenant migration guide with the following changes:
* Update the status requirements saying that the only payments/submitter_transactions statuses that need to be solved before the migration are PENDING/PROCESSING
* Reorganize the content
* Add extra sections with more info
* Include new commands and SQL queries to make the migration easier.
* Add TODOs:
  - [x] https://github.com/stellar/stellar-disbursement-platform-backend/pull/262
  - [ ] https://github.com/stellar/stellar-disbursement-platform-backend/pull/266
  - [ ] https://stellarorg.atlassian.net/browse/SDP-1148

### CODE REVIEW

File: network/stellar-disbursement-platform/admin-guide/migrating-to-sdp-multi-tenant.mdx [hunk 1]
```diff
@@ -1,89 +1,221 @@
...
 - `ENABLE_SCHEDULER`: Default "false". This enables scheduled jobs that replace the operations of a broker for synchronizing payments between SDP and TSS as well as submitting receiver invitations.
 - `SCHEDULER_PAYMENT_JOB_SECONDS`: Interval in seconds for the job that synchronizes payments between SDP and TSS. Minimum is 5s.
 - `SCHEDULER_RECEIVER_INVITATION_JOB_SECONDS`: Interval in seconds for the job that submits receiver invitations. Minimum is 5s.
 
 On the Anchor Platform side, we must set the following envs:
 
-- `SEP10_HOME_DOMAINS`: "localhost:8000, *.stellar.local:8000" # Comma separated list of home domains
-- `SEP10_HOME_DOMAIN`: "" # Here should be empty for the Multi-tenant version
+- `SEP10_HOME_DOMAINS="localhost:8000, *.stellar.local:8000"`: a comma-separated list of home domains used for [SEP-10].
+- `SEP10_HOME_DOMAIN=""`: this should be an empty string for the Multi-tenant version.
```
Comment (marwen-abid) [inline network/stellar-disbursement-platform/admin-guide/migrating-to-sdp-multi-tenant.mdx:137]: I believe we need to add `SEP10_WEB_AUTH_DOMAIN` here

File: network/stellar-disbursement-platform/admin-guide/migrating-to-sdp-multi-tenant.mdx [hunk 2]
```diff
@@ -1,89 +1,221 @@
...
+- `KAFKA_SSL_ACCESS_KEY`: The Kafka Access Key (keystore) in PEM format, required when the kafka security protocol is set to `SSL`.
+- `KAFKA_SSL_ACCESS_CERTIFICATE`: The Kafka SSL Access Certificate in PEM format that matches with the Kafka Access Key, required when the kafka security protocol is set to `SSL`.
+
+Scheduler environment variables:
 
 - `ENABLE_SCHEDULER`: Default "false". This enables scheduled jobs that replace the operations of a broker for synchronizing payments between SDP and TSS as well as submitting receiver invitations.
 - `SCHEDULER_PAYMENT_JOB_SECONDS`: Interval in seconds for the job that synchronizes payments between SDP and TSS. Minimum is 5s.
 - `SCHEDULER_RECEIVER_INVITATION_JOB_SECONDS`: Interval in seconds for the job that submits receiver invitations. Minimum is 5s.
 
 On the Anchor Platform side, we must set the following envs:
```
Comment (marwen-abid) [inline network/stellar-disbursement-platform/admin-guide/migrating-to-sdp-multi-tenant.mdx:134]: Should we also mention the version compatibility of Anchor ? I believe MTN is only compatible with `2.6.0`+ as the `SEP10_HOME_DOMAINS` variable wasn't available before.

**Review by marwen-abid — APPROVED** [review]
LGTM! 🚀 These changes are amazing! Thank you for all the updates.

File: network/stellar-disbursement-platform/admin-guide/migrating-to-sdp-multi-tenant.mdx (hunk 2 above)
Comment (marcelosalloum) [reply network/stellar-disbursement-platform/admin-guide/migrating-to-sdp-multi-tenant.mdx:142]: Addressed in https://github.com/stellar/stellar-docs/pull/496/commits/143409eb702fceebac7b7cf0b6c744e10ccd3a14

File: network/stellar-disbursement-platform/admin-guide/migrating-to-sdp-multi-tenant.mdx (hunk 1 above)
Comment (marcelosalloum) [reply network/stellar-disbursement-platform/admin-guide/migrating-to-sdp-multi-tenant.mdx:145]: Addressed in https://github.com/stellar/stellar-docs/pull/496/commits/143409eb702fceebac7b7cf0b6c744e10ccd3a14

**Pull request merged**